### ISSUE TITLE
Layer doesn't show, doesn't throw error

### ISSUE THREAD

**Issue by ra100** [maptalks/maptalks.odline#2]
I'm trying to make ODLine work inside vue, but nothing works.
I'm initializing it in store 
```js
import {ODLineLayer} from 'maptalks.odline'
...
layerTrails: new ODLineLayer('trails', []),
```
[link](https://github.com/ra100/carshare-exporter-viz/blob/a63fb14db7329024ee632f47909d6f2744371794/src/store/index.js#L26)
adding it to map `.addTo(state.map)` and then set data:
`state.layerTrails.setData(state.trails)` [link](https://github.com/ra100/carshare-exporter-viz/blob/a63fb14db7329024ee632f47909d6f2744371794/src/store/mutations.js#L42)
I don't get any error, and layer doesn't show either.
I there anything I should look for?

**Comment by fuzhenn** [comment]
Hey, @ra100 , excited to hear from you.

This is probably because odline is a little bit outdated and I forgot to upgrade it to [ES module](https://github.com/rollup/rollup/wiki/pkg.module) as other plugins.

I have just [updated odline to ES modules](https://github.com/maptalks/maptalks.odline/commit/4d5f4822d89b2fb869eb0ce05cbcf229adf5b53e) and re-published it to npm, you can try to upgrade odline to v0.4.0 and see if it resolves your problem.

**Comment by ra100** [comment]
Hi @fuzhenn,
thanks for quick answer, I updated to 0.4.0, but there is no change.
Still no error, and no display. I must be missing something.

This is how odline layer looks like, it looks OK.
![54a717ba-9804-49aa-81cb-d953e8e51f26](https://user-images.githubusercontent.com/3185410/31853747-3be30a6c-b68e-11e7-88d6-01f2da8a7690.png)

**Comment by fuzhenn** [comment]
Alright, this is a little bit confusing. 
Could you please provide a minimal reproducing example to me?
Upload here or in any jsfiddle like host.

**Comment by ra100** [comment]
OK, It will take some time, but I'll do my best.

**Comment by ra100** [comment]
Here is the example: https://jsfiddle.net/sk01gttm/17/
trails showed there once, but I couldn't replicate conditions when it worked. I tried different combinations:
```
const ol = new maptalks.ODLineLayer('trails', [])
const ol = new maptalks.ODLineLayer('trails', [], {})
const ol = new maptalks.ODLineLayer('trails', []).addTo(map)
```
and so on

EDIT: seems it works with {animation: false}

**Comment by ra100** [comment]
OK, I think I found problem, partially: https://jsfiddle.net/ra100/sk01gttm/18/
when I do not set `globalCompositeOperation`, the default option doesn't work/show, if I put there null, it works, and with some values from here too: https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/globalCompositeOperation
these seems not working: `source-in`, `source-out`, `source-atop`, `destination-in`, `destination-out`, `destination-atop`, `copy`

**Comment by fuzhenn** [comment]
Great!
Default `globalCompositeOperation` shouldn't show nothing anyway, I'll look into this later.

**Comment by fuzhenn** [comment]
@ra100 
fixed and published as v0.4.1

**Comment by ra100** [comment]
great, it works
